config_converter/__main__.py

```diff
@@ -0,0 +1,4 @@
+from config_converter import config_converter
+
+if __name__ == '__main__':
```

Comment:
I think we can combine __main__.py into config_converter.py. An example is https://github.com/googleapis/google-cloud-python/blob/master/appveyor/env_info.py#L29-L48

Follow style in for the main function: https://google.github.io/styleguide/pyguide.html#317-main

Reply:
This definitely makes it cleaner - will make that change!